src/app/app.css

```diff
@@ -23,3 +23,8 @@ html, body, #root, .app  {
   display: flex;
   flex-direction: column;
 }
+
+body  {
+  font-family: "Tahoma", sans-serif;
```

Comment:
Is it the same font that's used by windows 10?

Reply:
I don't know the exact font used in W10, so I chose "Tahoma" for now.

Reply:
Then research time :P